### DISCUSSION TITLE
Installing Snipe-IT in Docker

### DISCUSSION THREAD

**Discussion by jmtechy** [snipe/snipe-it#9166] [Q&A]
Hello all, 

I am attempting to install Snipe-it onto Docker following instructions [here](https://snipe-it.readme.io/docs/docker). However am having the following issue:

After I run the "docker run --rm snipe/snipe-it" command and it returns the APP_Key as listed below:

"Please re-run this container with an environment variable $APP_KEY
An example APP_KEY you could use is: 
base64:D5oGA+zhFSVA3VwuoZoQ21RAcwBtJv/RGiqOcZ7BUvI="

I then grab the entire APP Key provided including the "base64:" and place it on MY_ENV_File where I had initially left the place holder. However, when I re-run the command "docker run --rm snipe/snipe-it" I get the same return with different App Key.

My question is, am I placing the App key on the correct file? 
if the .ENV file is not the same as MY_ENV_File then, where can I find the .ENV file referenced in the guide listed on link above? I have been trying to identify the .ENV file but am not able to find it anywhere.

Any help please will be much appreciated.

Thanks in advanced

**Comment by uberbrady** [comment]
The `APP_KEY` is supposed to be random, so every time you run it you get a new value.

Whatever you're calling your Docker env-file - `MY_ENV_File` or whatever - the APP_KEY should go in there. Once you do that and re-run the snipe-it container with the `--env-file=MY_ENV_File` - it should not complain about a missing `APP_KEY`

**Reply by jmtechy** [reply]
Thank you for the reply. 

The problem is that it continues to complain. Any suggestions, or any more info I can provide for help?
